catalogue/webapp/services/wellcome/catalogue/types/index.ts

```diff
@@ -112,13 +112,15 @@ type ConceptType =
   | 'Person'
   | 'Concept'
   | 'Period'
-  | 'Place';
+  | 'Place'
+  | 'Genre';
 
 export type Concept = {
   id?: string;
   identifiers?: Identifier[];
   label: string;
   type: ConceptType;
+  sameAs?: string[];
```

Comment:
For my own knowledge, is sameAs just a list of same concepts? Like "Glasgow the city" and "Glasgow the dark organisation of a dystopian future"?

Reply:
This one's definitely a relevance thing for @harrisonpim to comment on.

Reply:
Yes. We have some concepts that exist as multiple Concepts because:

1. type is used as part of the key when generating canonical ids
2. type is estimated from the field in the data from the source system

This is particularly true of Genres because there is no way of knowing that a thing is a genre when it is a subject, so we would not be able to find both about and using in the same page.  The Concepts pipeline reconciles them to make the sameAs list.

It's a more reliable solution to the Glasgow Problem than the previous one, as it now works across documents, instead of just in the context of a single one.